src/templates/components/chapter/CodingInterface.js

```diff
@@ -76,11 +76,12 @@ const CodingInterface = ({
           <button className={`mr-6`} onClick={resetEditorCode}>
             <RefreshIcon />
           </button>
-          <button
+          <a
```

Comment:
Please add target="_blank" & rel="noopener" attributes for opening link in a new tab for better UX.